index.html

```diff
@@ -86,6 +86,13 @@
           status:   "Internet-Draft",
           publisher:  "IETF"
         },
+        "SD-JWT VC": {
```

Comment:
I don't think you can have a space in a reference tag.  Maybe change the space to dash.